nextcord/state.py

```diff
@@ -267,6 +270,7 @@ def __init__(
         ] = {}
         # A dictionary of Discord Application Command ID's and the ApplicationCommand object they correspond to.
         self._application_command_ids: Dict[int, BaseApplicationCommand] = {}
+        self._automod_rules: dict[int, AutoModerationRule] = {}
```

Comment:
Why did you define this twice here and in `clear`? Near the bottom of this `__init__`, `clear` is called anyways.

Reply:
for consistency. 

imo clear could be used for more thing so... add it to prevent

Reply:
But it's redundant